packages/react/src/components/Button/Button.stories.js

```diff
@@ -182,3 +188,44 @@ export const Skeleton = () => {
     </div>
   );
 };
+
+export const SetOfButtonsFluid = {
+  parameters: {
+    controls: {
+      include: ['Fluid Buttons', 'Stacked'],
+    },
+  },
+  argTypes: {
+    'Fluid Buttons': {
+      control: {
+        type: 'select',
+        labels: fluidButtonLabels,
+      },
+      options: fluidButtonOptions,
+      mapping: fluidButtonMapping,
+    },
+  },
+  render: ({ ...rest }) => {
+    const buttons = rest['Fluid Buttons'];
+
+    if (!buttons || buttons === 0) {
+      return <div>Select one or more buttons.</div>;
+    }
+
+    return (
+      <WithDisplayBox>
```

Comment:
I don't think we want to go this route unfortunately. It didn't pass a visual design review and it's not easily testable with VRT.

Maybe we talked about this before, but I think a decent alternative is having a `containerWidth` control that explicitly states in the description that it's just for testing in storybook. 

We can then set this control via args in VRT tests to ensure this remains consistent at different sizes with different button counts. The other controls that aren't part of the component api could also have this note in their description.